packages/p2p/src/components/buy-sell/filter-modal/filter-modal.scss

```diff
@@ -53,22 +53,15 @@
     }
 
     &__toggle {
-        align-items: center;
-        display: flex;
-        justify-items: flex-start;
-        transform: scale(0.5) translateX(-50%);
-
         &-button {
-            top: 3px;
-
-            &--on {
-                left: calc(100% - 4px);
-            }
+            margin-top: -1px;
         }
 
         &-label {
-            height: 2.4rem;
-            width: 4.5rem;
+            margin: 0 auto;
+            @media (max-width: 375px) {
+                margin: 0;
+            }
```

Comment:
can we use `@include mobile` instead of a custom media query.

Reply:
thanks for this remark. 
I checked it with`@include mobile` and it seems we don’t need here adjust margin with special query. For this window `margin: 0` works in any width.
Just made  `margin: 0`